app/models/patient.rb

```diff
@@ -90,6 +91,11 @@ class Patient < ApplicationRecord
             absence: { message: "cannot be 'true' when 'Last Date of Exposure' is specified" },
             if: -> { last_date_of_exposure.present? }
 
+  validates :follow_up_note,
+            on: %i[api import],
+            absence: { message: "must be blank when 'Follow-Up Reason' is blank" },
+            if: -> { follow_up_reason.nil? }
```

Comment:
Thanks for adding this, could you just test this in the `test/models/patient_test.rb` file? There is a big chunk of tests testing other validations, so it can fit somewhere in there. And a test for `follow_up_reason` as well.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/983/commits/77f00816d1ddaebf1af47d9051ce5c8c0c6a268f